src/pages/TeamSettings/Service-Accounts.vue

```diff
@@ -61,8 +61,13 @@ export default {
     filteredRoles() {
       if (!this.roles) return []
       let rolesToRM = ['RUNNER']
-      if (!this.hasPermission('license', 'admin')) {
```

Comment:
Sorry I think I gave bad advice when you checked this with me yesterday - we will want to check for both license:admin to see if we should show enterprise license admin and then also check for feature:basic-rbac to see if we should show user and read only (and should remove runner for all).